tools/apigee-x-trial-provision/README.md

```diff
@@ -0,0 +1,10 @@
+# Apigee X Trial Provisioning
+
+> Note - this script is not subject to pipelines
+
+## Description
+
+A reference example of [Command Line](https://cloud.google.com/apigee/docs/api-platform/get-started/install-cli)
+Provisioning for Apigee X.
+
```

Comment:
Maybe include list of overridable default values in here?

Reply:
fixed and included caveat. this is mainly for reference and will be improved in future (variable validation, proper certs etc.)